src/components/SelectionList/Search/ReportInfoCell.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Avatar from '@components/Avatar';
+import Text from '@components/Text';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {getPolicy} from '@libs/PolicyUtils';
+import {getIcons, getReportName, getReportOrDraftReport} from '@libs/ReportUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+type ReportInfoCellProps = {
+    reportID: string;
+};
+
+function ReportInfoCell({reportID}: ReportInfoCellProps) {
+    const styles = useThemeStyles();
+    const {isLargeScreenWidth} = useResponsiveLayout();
+    const [personalDetails] = useOnyx(ONYXKEYS.PERSONAL_DETAILS_LIST);
+
+    const report = getReportOrDraftReport(reportID);
+
+    if (!report || !personalDetails) {
+        return null;
+    }
+
+    const policy = getPolicy(report.policyID);
+    const reportName = getReportName(report, policy, undefined, undefined);
+    const icons = getIcons(report, personalDetails, null, '', -1, policy);
+    const icon = icons?.at(0);
```

Comment:
I think we should make this as much of a pure component as we can, meaning passing this info from the item instead of fetching it inside the component. That would improve reusability